.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -311,6 +311,8 @@ jobs:
           os: [ubuntu-22.04]
           suite:
             - blog-admin-tests/assign-roles-to-users-and-change-tag-properties.spec.js
+            - exploration-editor-tests/exploration-creator-manager.spec.js
```

Comment:
Filename does not follow the way other filenames are phrased. Please fix.

Ditto below.

Reply:
@seanlip  .I am little bit confused about the name. As per `doc` 
![image](https://github.com/oppia/oppia/assets/136263179/22c91cfe-3bf3-4bcc-a2a3-2fdd50817c67)
 user journey description i think it can be `exploration-creator-tests/create-exploration-and-change-basic-settings.spec.js` . I would appreciate any suggestion you can give me.

Reply:
@seanlip PTAL.(Thanks) :•)

Reply:
The first title seems fine. I don't know what "publish draft discard changes" means in the second one, but I think you should be able to edit it to be clearer. It should read like a standard English sentence/phrase.

Reply:
`exploration-creator-tests/savedraft-publish-and-discard-the-changes.spec.js` @seanlip PTAl for second one now.
IMO its look more clear now. As in the user journey we are doing some changes and then savedraft those changes,publish those changes and then discard those changes.

Reply:
updated ✅

Reply:
@rahat2134 I don't see your code changes in the files changed tab. It still says exploration-creator-manager.spec.js for me.

I'm going to stop reviewing for now -- make sure you've pushed all the changes and that the Files Changed tab matches what you want reviewers to look at.

Also please note that the acceptance tests were recently updated to use TypeScript, please use typescript for your PR as well. Thanks!